app/BUILD.bazel

```diff
@@ -10,23 +10,26 @@ order to build with Bazel.
 - All binding adapters must be written in Java.
 """
 
-load("@rules_jvm_external//:defs.bzl", "artifact")
 load("@dagger//:workspace_defs.bzl", "dagger_rules")
+load("@io_bazel_rules_kotlin//kotlin:kotlin.bzl", "kt_android_library")
+load("@rules_android//android:rules.bzl", "android_library")
+load("@rules_jvm_external//:defs.bzl", "artifact")
 load("@tools_android//tools/crashlytics:defs.bzl", "crashlytics_android_library")
 load("@tools_android//tools/googleservices:defs.bzl", "google_services_xml")
-load("@io_bazel_rules_kotlin//kotlin:kotlin.bzl", "kt_android_library")
 load("//app:app_test.bzl", "app_test")
 load("//app:test_with_resources.bzl", "test_with_resources")
 
-"""
-Source files for the annotations library. The files inside the annotations library are dependencies
-for all other libraries in app module, except the listeners library.
-Place your file here if:
- - A view, view model, or layout file depends on it
- - It is an annotation or helper file of some kind
- - It is not considered a listener
- - It does not depend on any other file not included in this list
-"""
+package(default_visibility = ["//utility:__subpackages__"])
+
+exports_files(["src/main/AndroidManifest.xml"])
```

Comment:
@vinitamurthi as per your earlier comment 

```
I think we should keep package(default_visibility) and remove these. But the package default visibility should not be //visibility:public
Instead it should be a list of packages that have this as a dependency (or private in case its all private)
```

Reply:
The default visibility should specifically be private here & elsewhere.

Reply:
Please resolve this, also. We shouldn't be setting package-level visibility anywhere (since the default is private).